### PR TITLE
Janitor Carbon Rewrite

### PR DESCRIPTION
**Pull request by Xanderxx46**
Rewrites janitor to use [Carbon](https://github.com/buape/carbon) instead. 

Ive done my own testing and as long as you do the install stuff right and deploy and shit youll be good. 

(If you see any bugs lmk, but it should work fine)

I didnt know how to do another way so thats why all files were deleted then reuploaded lol

### CODE REVIEW

File: src/commands/werd.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Command, type CommandInteraction } from "@buape/carbon";
+
+export default class englishCommand extends Command {
```
Comment (thewilloftheshadow) [inline src/commands/werd.ts]: This command can be deleted entirely

File: README.md [hunk 2]
```diff
@@ -1,45 +1,45 @@
-# JT99-Janitor
```
Comment (thewilloftheshadow) [inline README.md:1]: I'd like to leave this readme file, edit it as needed for the new version

**Review by thewilloftheshadow — CHANGES_REQUESTED** [review]

File: wrangler.toml [hunk 3]
```diff
@@ -1,7 +1,3 @@
-name = "jt99-janitor"
-main = "src/index.ts"
-account_id = "07c634e5b9f9eb260972946dc8e2f891"
-compatibility_date = "2022-09-03"
-
-[build]
-    command = "npm run build"
\ No newline at end of file
+name = "jt99-janitor-rewrite"
```
Comment (thewilloftheshadow) [inline wrangler.toml]: You can keep the name the same, we don't need to keep the old worker around
```suggestion
name = "jt99-janitor"
```

File: README.md (hunk 2 above)
Comment (Xanderxx46) [reply README.md:1]: Its the exact same i just deleted and reuploaded it (also idk how to undo deletes)

**Review by thewilloftheshadow — APPROVED** [review]

**Pull request merged**